spec/active_record/connection_adapters/makara_postgresql_adapter_spec.rb

```diff
@@ -73,7 +73,13 @@
       Test::User.exists? # flush other (schema) things that need to happen
 
       con = connection.replica_pool.connections.first
-      expect(con).to receive(:exec_query).with(/SELECT\s+1\s*(AS one)?\s+FROM .?users.?\s+LIMIT\s+.?1/, any_args).once.and_call_original
+
+      expect(con).to receive(:exec_query) do |query|
```

Comment:
Seems like this change is causing a failure for this matrix run: https://github.com/instacart/makara/pull/283/checks?check_run_id=1730065278

Reply:
No, this spec flaps on jruby for some reason. It's the same on master now.

Reply:
See https://github.com/instacart/makara/runs/1705529911